scripts/demangle.py

```diff
@@ -31,24 +46,27 @@ def extract_func_code(func_name, file_path):
         func_now += 1
 
     if open_braces > 0: # raise ValueError(f"Malformed function definition for '{func_name}' in code file")
-        print(f"ERROR: Malformed function definition for '{func_name}' in code file")
-        return f"ERROR: Malformed function definition for '{func_name}' in code file"
+        print(f"ERROR: Malformed function definition for '{ori_func_name}' in code file")
+        return f"ERROR: Malformed function definition for '{ori_func_name}' in code file"
 
     function_code = code_content[func_start:func_now]
     return func_init+function_code
 
-def demangle(func: str):
+def demangle_func(func: str):
     # Load the C++ Standard Library
     libcxx = ctypes.cdll.LoadLibrary("libc++.so.1")
-
     # Define the signature of the __cxa_demangle() function
     demangle = libcxx.__cxa_demangle
     demangle.argtypes = [ctypes.c_char_p, ctypes.c_char_p, ctypes.POINTER(ctypes.c_size_t), ctypes.POINTER(ctypes.c_int)]
     # Call __cxa_demangle() to demangle the input string
     output_buffer = ctypes.create_string_buffer(256)
     output_size = ctypes.c_size_t(256)
     status = ctypes.c_int(0)
+    if cnt>=66:
```

Comment:
Why 66?

Reply:
It's just a debug print, I have deleted it in the new version.